db/seeds.rb

```diff
@@ -13,4 +13,11 @@
 FactoryGirl.create(:section, name: "3")
 FactoryGirl.create(:section, name: "4")
 
-puts " Successfully added groups from seed data. ".center(70, '~*~')
+puts " Successfully added sections from seed data. ".center(70, '~*~')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.